pkgs/top-level/emacs-packages.nix

```diff
@@ -169,7 +169,7 @@ let
     # already byte-compiled by Agda builder
     buildPhase = ''
       agda=`${Agda}/bin/agda-mode locate`
-      cp `dirname $agda`/*.el* .
+      cp `dirname $agda`/*.el* . # */
```

Comment:
I also do that to avoid syntax highlighting problem in Emacs, but I don't commit it usually :-).